test/hooks_functional.js

```diff
@@ -16,13 +16,16 @@ import events from 'events';
 
 events.EventEmitter.prototype._maxListeners = 100;
 
+const timeout = 50000;
```

Comment:
Since this a global constant, could you please capitalize it?
```suggestion
const TIMEOUT = 50000;
```